pages/having-a-child.js

```diff
@@ -1,10 +1,18 @@
 import Layout from "../components/layout";
+import MoreInfo from "../components/atoms/MoreInfo";
 
 export default function lifejourney() {
   return (
     <Layout
       bannerTitle="Having a child"
       bannerText="Lorem ipsum dolor sit amet, consectetur"
-    ></Layout>
+    >
+      <div>
```

Comment:
maybe we could start now including translation placeholders instead of hardcoded text?

Reply:
we'll chat about this ...  need to see how we're gonna do text in the end

Reply:
ignore my comments!  different approach